packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -9,6 +9,8 @@
               v-if="logo"
               :src="logo"
               :alt="title"
+              :width="34"
```

Comment:
This indeed solves the problem but maybe it would be a good opportunity to introduce two new props to SfHeader? In this component we accept a prop of logo that is being passed to the SfImage component. So instead of hardcoding the sizes like that we can create props with default values so that when user passes their own props the size of logo would be changed without using CSS.

```vue
props: {
  logoHeight: {
   type: Number,
   default: 35
  } ...
}
```

And then here

```suggestion
              :width="logoWidth"
              :height="logoHeight"
```